2024-10-20 21:35:53.801 - RealTimeSTT: root - INFO - Starting RealTimeSTT
2024-10-20 21:35:53.811 - RealTimeSTT: root - INFO - Initializing audio recording (creating pyAudio input stream, sample rate: 16000 buffer size: 512
2024-10-20 21:35:53.815 - RealTimeSTT: root - INFO - Initializing WebRTC voice with Sensitivity 3
2024-10-20 21:35:53.815 - RealTimeSTT: root - DEBUG - WebRTC VAD voice activity detection engine initialized successfully
2024-10-20 21:35:56.588 - RealTimeSTT: torio._extension.utils - DEBUG - Loading FFmpeg6
2024-10-20 21:35:57.260 - RealTimeSTT: torio._extension.utils - DEBUG - Failed to load FFmpeg6 extension.
Traceback (most recent call last):
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torio\_extension\utils.py", line 116, in _find_ffmpeg_extension
    ext = _find_versionsed_ffmpeg_extension(ffmpeg_ver)
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torio\_extension\utils.py", line 108, in _find_versionsed_ffmpeg_extension
    _load_lib(lib)
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torio\_extension\utils.py", line 94, in _load_lib
    torch.ops.load_library(path)
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torch\_ops.py", line 1032, in load_library
    ctypes.CDLL(path)
  File "C:\Users\kapil\AppData\Local\Programs\Python\Python310\lib\ctypes\__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'C:\Users\kapil\Downloads\html\.venv\Lib\site-packages\torio\lib\libtorio_ffmpeg6.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
2024-10-20 21:35:57.267 - RealTimeSTT: torio._extension.utils - DEBUG - Loading FFmpeg5
2024-10-20 21:35:57.976 - RealTimeSTT: torio._extension.utils - DEBUG - Failed to load FFmpeg5 extension.
Traceback (most recent call last):
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torio\_extension\utils.py", line 116, in _find_ffmpeg_extension
    ext = _find_versionsed_ffmpeg_extension(ffmpeg_ver)
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torio\_extension\utils.py", line 108, in _find_versionsed_ffmpeg_extension
    _load_lib(lib)
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torio\_extension\utils.py", line 94, in _load_lib
    torch.ops.load_library(path)
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torch\_ops.py", line 1032, in load_library
    ctypes.CDLL(path)
  File "C:\Users\kapil\AppData\Local\Programs\Python\Python310\lib\ctypes\__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'C:\Users\kapil\Downloads\html\.venv\Lib\site-packages\torio\lib\libtorio_ffmpeg5.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
2024-10-20 21:35:57.976 - RealTimeSTT: torio._extension.utils - DEBUG - Loading FFmpeg4
2024-10-20 21:35:58.475 - RealTimeSTT: torio._extension.utils - DEBUG - Failed to load FFmpeg4 extension.
Traceback (most recent call last):
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torio\_extension\utils.py", line 116, in _find_ffmpeg_extension
    ext = _find_versionsed_ffmpeg_extension(ffmpeg_ver)
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torio\_extension\utils.py", line 108, in _find_versionsed_ffmpeg_extension
    _load_lib(lib)
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torio\_extension\utils.py", line 94, in _load_lib
    torch.ops.load_library(path)
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torch\_ops.py", line 1032, in load_library
    ctypes.CDLL(path)
  File "C:\Users\kapil\AppData\Local\Programs\Python\Python310\lib\ctypes\__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'C:\Users\kapil\Downloads\html\.venv\Lib\site-packages\torio\lib\libtorio_ffmpeg4.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
2024-10-20 21:35:58.475 - RealTimeSTT: torio._extension.utils - DEBUG - Loading FFmpeg
2024-10-20 21:35:58.475 - RealTimeSTT: torio._extension.utils - DEBUG - Failed to load FFmpeg extension.
Traceback (most recent call last):
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torio\_extension\utils.py", line 116, in _find_ffmpeg_extension
    ext = _find_versionsed_ffmpeg_extension(ffmpeg_ver)
  File "C:\Users\kapil\Downloads\html\.venv\lib\site-packages\torio\_extension\utils.py", line 106, in _find_versionsed_ffmpeg_extension
    raise RuntimeError(f"FFmpeg{version} extension is not available.")
RuntimeError: FFmpeg extension is not available.
2024-10-20 21:35:58.558 - RealTimeSTT: root - DEBUG - Silero VAD voice activity detection engine initialized successfully
2024-10-20 21:35:58.559 - RealTimeSTT: root - DEBUG - Starting realtime worker
2024-10-20 21:35:58.559 - RealTimeSTT: root - DEBUG - Waiting for main transcription model to start
2024-10-20 21:35:58.560 - RealTimeSTT: root - DEBUG - Main transcription model ready
2024-10-20 21:35:58.560 - RealTimeSTT: root - DEBUG - RealtimeSTT initialization completed successfully
2024-10-20 21:35:58.560 - RealTimeSTT: root - INFO - recording started
2024-10-20 21:35:58.560 - RealTimeSTT: root - INFO - State changed from 'inactive' to 'recording'
2024-10-20 21:35:58.560 - RealTimeSTT: root - INFO - State changed from 'recording' to 'inactive'
2024-10-20 21:35:58.561 - RealTimeSTT: root - INFO - Attempted to stop recording too soon after starting.
2024-10-20 21:35:59.263 - RealTimeSTT: werkzeug - INFO - 127.0.0.1 - - [20/Oct/2024 21:35:59] "[35m[1mPOST /audio_scam_check HTTP/1.1[0m" 500 -
